CHANGELOG.yaml

```diff
@@ -1,263 +1,16 @@
-  new features:
-    - 'Added options to provide client-side SSL certificates on the command line'
-    - 'Updated the versions of postman-runtime and postman-collection libraries'
-
-3.2.0:
-  date: 2016-10-18
-  new features:
-    - 'Ensure that environment exports are consistent with the app'
-    - 'Simplified the importing logic for environment and globals'
-
-3.1.2:
-  date: 2016-08-25
-  fixed bugs:
-    - 'Updated postman-runtime, postman-collection and postman-collection-transformer dependencies'
-
-3.1.1:
-  date: 2016-08-25
-  fixed bugs:
-    - 'Fixed a bug which caused HTML reporter statistics to be incorrect'
-    - 'Added an option --disable-unicode to forcibly disable unicode symbols in the output'
-    - 'File based options in newman.run (environments, data, globals) can now be specified as JSON objects as well'
-    - 'Updated to the latest version of Postman Runtime, which contains a number of memory optimizations'
-    - 'Specifying an iteration data file now results in the correct number of iteration runs'
-
-3.1.0:
-  date: 2016-08-25
-  new features:
-    - 'Fixed issue with environment and globals export format was using wrong property names (GH:553)'
-    - 'Fixed issue where --export-* CLI option did not work with no parameters'
-    - 'Added support for postman.getResponseCookie in the script sandbox'
-    - 'Prettified HTML report template'
-    - 'Added support for custom HTML report templates via --reporter-html-template flag'
-    - 'Added file upload support'
-    - 'Fixed a bug in the timeout request flag: #547'
-    - 'Updated runtime dependency to 2.4.4'
-    - 'Fixed a bug that caused incorrect iterationCount detection'
-    - 'Better error messages'
-    - 'Fixed a bug that caused incorrect exports of environment and global values'
-    - 'The export parameters now default to /newman/*'
-    - 'Added a new CLI option, --silent which ensures Newman does not write anything to the terminal'
-
-3.0.1:
-  date: 2016-08-09
-  fixed bugs:
-    - 'Updated Postman Runtime to v2.4.1'
-    - 'Newman now exits with a non-zero status code on test failures or errors'
-
-3.0.0:
-  date: 2016-08-09
-  fixed bugs:
-    - 'First stable release of Newman v3.0'
-    - 'Added ability to suppress exit code on failures'
-    - 'Renamed the "--stop-on-error" option to "--bail", which ensures that Newman stops on test failures or errors'
-
-3.0.0-rc.1:
-  date: 2016-08-08
-  initial release:
-    - 'Initial release of Newman v3. Migration Guide'
+unreleased:
+  new features:
+    - '#1633 Added ability to fetch resources via Postman API UID reference'
+  breaking changes:
+    - '#1605 Dropped support for Node v4'
+    - '#1610 Dropped support for v2 CLI options'
+    - '#1616 Moved inbuilt HTML reporter to standalone reporter: `newman-reporter-html`'
+  fixed bugs:
+    - '#1609 Fixed CSV auto parse, to avoid parsing numbers inside quotes'
+    - '#1630 Updated default timeout values to Infinity'
+    - '#1623 Refactored Newman CLI'
```

Comment:
This and the section below should go into a section called `chores` since they aren't exactly bugs. (Confirm that this won't cause Yankee to break)